server/config/config.go

```diff
@@ -105,9 +105,10 @@ type S3CacheConfig struct {
 }
 
 type DistributedCacheConfig struct {
-	ListenAddr  string `yaml:"listen_addr" usage:"The address to listen for local BuildBuddy distributed cache traffic on."`
-	RedisTarget string `yaml:"redis_target" usage:"A redis target for improved Caching/RBE performance. ** Enterprise only **"`
-	GroupName   string `yaml:"group_name" usage:"A unique name for this distributed cache group. ** Enterprise only **"`
+	ListenAddr        string `yaml:"listen_addr" usage:"The address to listen for local BuildBuddy distributed cache traffic on."`
+	RedisTarget       string `yaml:"redis_target" usage:"A redis target for improved Caching/RBE performance. ** Enterprise only **"`
+	GroupName         string `yaml:"group_name" usage:"A unique name for this distributed cache group. ** Enterprise only **"`
+	ReplicationFactor int    `yaml:"replication_factor" usage:"How many servers the data should be replicated to. ** Enterprise only **"`
```

Comment:
nit: document whether ReplicationFactor accounts for the local server

Reply:
Done.